### PR TITLE
load group name from plist

### PR DESCRIPTION
**Pull request by jishnu7**

### CODE REVIEW

File: ios/UtilsPlugin.mm [hunk 1]
```diff
@@ -33,7 +33,9 @@ - (void) performActionForShortcutItem:(UIApplicationShortcutItem *) shortcutItem
 }
 
 - (void) updateShortcutItems: (NSDictionary *)jsonObject {
-    NSUserDefaults *sharedDefaults = [[NSUserDefaults alloc] initWithSuiteName:@"group.com.hashcube.sudokuquest"];
+    NSString *appGroup = [[NSBundle mainBundle] objectForInfoDictionaryKey:@"WidgetGroup"];
```
Comment (rampr) [inline ios/UtilsPlugin.mm]: shouldn't we take it from the iOS section ?

Comment (jishnu7) [inline ios/UtilsPlugin.mm]: this is reading from the Info.plist

Comment (rampr) [inline ios/UtilsPlugin.mm]: okay

Comment (Bijoshtj) [inline ios/UtilsPlugin.mm]: what does this widgetGroup represents?

Comment (jishnu7) [inline ios/UtilsPlugin.mm]: groups are needed to share data between apps. so, if you want to share data between a widget and an app, you need both to be in same group

**Pull request merged**